### PR TITLE
infra/gcp/prow: use two default ssh keys

### PR DESCRIPTION
**Pull request by spiffxp**
This is followup to https://github.com/kubernetes/k8s.io/pull/1894

Fixes https://github.com/kubernetes/k8s.io/issues/1751

I would rather just use the one, but I no longer have time to untangle whatever is going on between prow, kubetest2, cluster/log-dump.sh and `gcloud ssh` (ref: https://github.com/kubernetes/k8s.io/issues/1751#issuecomment-816835097)

Since the steady state we've converged on is most projects having both keys, make that the default / expected state

As with https://github.com/kubernetes/k8s.io/pull/1894 I've added a [wip] commit to this PR to make running this script faster.  Will remove commit and [wip] from PR once actuated.

### CODE REVIEW

**Comment by spiffxp** [comment]
OK, removed [wip] commit https://github.com/kubernetes/k8s.io/commit/712053a71a391f7894041e3a7be2a4beab5fa37f

Ran `K8S_INFRA_ENSURE_E2E_PROJECTS_RESETS_SSH_KEYS=true ./ensure-e2e-projects.sh`, added keys to:
- k8s-infra-e2e-gce-project
- k8s-infra-e2e-node-e2e-project
- k8s-infra-e2e-boskos-scale-12

**Review by hasheddan — APPROVED** [review]
/lgtm

File: infra/gcp/prow/ensure-e2e-projects.sh [hunk 1]
```diff
@@ -165,24 +165,35 @@ for prj; do
...
+      for ssh_key in "${ssh_keys_expected[@]}"; do
+        if ! grep -q "${ssh_key}" "${ssh_keys_before}"; then
+          echo "${ssh_key}" >> "${ssh_keys_after}"
+        fi
+      done
     fi
 
     if ! diff ${ssh_keys_before} ${ssh_keys_after} >/dev/null; then
-      gcloud compute project-info add-metadata --project="${prj}" \
+      cloud compute project-info add-metadata --project="${prj}" \
```
Comment (ameukam) [inline infra/gcp/prow/ensure-e2e-projects.sh:196]: @spiffxp Nit?

Comment (spiffxp) [reply infra/gcp/prow/ensure-e2e-projects.sh:196]: good catch, thank you

Comment (spiffxp) [reply infra/gcp/prow/ensure-e2e-projects.sh:196]: https://github.com/kubernetes/k8s.io/pull/1907 should fix

**Pull request merged**